### ISSUE TITLE
Extend routing response with bearing for roundabout exits

### ISSUE THREAD

**Issue by Rungee** [GIScience/openrouteservice#25]
![image](https://user-images.githubusercontent.com/1660719/30648200-c5a8ad80-9e1d-11e7-8140-f6ee57a6e42e.png)

For the image above, the expected list of bearings will be [248,141,62].

The step element in routing instructions will have the following format:
```json
{
    "distance": 203.6,
    "duration": 23.6,
    "type": 7,
    "instruction": "Enter the roundabout and take the 2nd exit onto <b>Stralsunder Ring</b>",
    "name": "Stralsunder Ring",
    "exit_number": 2,
    "exit_bearings": [248, 141, 62],
    "way_points": [3, 21]
}
```

**Comment by Rungee** [comment]
Hi @farough!

What do you think about this kind of format for bearings of roundabout exits?

**Comment by Rungee** [comment]
This feature is implemented in 7513123949649a64f45a629d55fd22b299b63711.
